.github/workflows/test.yaml

```diff
@@ -22,6 +22,8 @@ jobs:
         # NOTE: checkout the code in a fixed location, even for forks, as this
         # is relevant for go's import system.
         path: ./src/github.com/snapcore/snapd
+    - name: Make /var/cache/apt owned by current user
+      run: sudo chown -R $(id -u) /var/cache/apt
```

Comment:
With this, we've got two steps doing a chown on the directory.  Do we actually need both?

I would have thought the ownership change was only necessary when `actions/cache@v1` is trying to extract a cache tarball: not when it is trying to tar up a new version of the cache.

Reply:
We need both because apt may fetch things and populate the cache.

Reply:
Sure, but does that prevent the action from tarring up the results?  The directory contents should still be world readable.

Reply:
Unfortunately it does. The `.lock` files there are not and this upset the action.

Reply:
Fair enough.